Sources/Request+Locale.swift

```diff
@@ -0,0 +1,45 @@
+import Vapor
+import Lingo
+
+extension Request {
+    var locale: String {
```

Comment:
Would be nice to document method and mention it in the README :)

Reply:
Done :)